src/chart/main/chart.cpp

```diff
@@ -147,6 +147,40 @@ void Chart::draw(Gfx::ICanvas &canvas)
 			    }
 		    });
 
+		actPlot->getOptions()->subtitle.visit(
+		    [this, &context](int, const auto &subtitle)
+		    {
+			    if (subtitle.value.has_value()) {
+				    Draw::DrawLabel(context,
+				        Geom::TransformedRect::fromRect(
+				            layout.subtitle),
+				        *subtitle.value,
+				        actPlot->getStyle().subtitle,
+				        events.draw.subtitle,
+				        std::make_unique<Events::Targets::ChartTitle>(
```

Comment:
We need a separate Events::Target Element for subtitle and footer with e corresponding tagName: subtitle, footer.

Reply:
OK

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/349/commits/3a55af09791c278ac56457adb4446af99e33cfb4